smtp-mail.cpp

```diff
@@ -152,5 +152,10 @@ int sendEmailMsg(const EmailCfg *emailCfg, const char *msg)
   delete upload_ctx.payload;
   return (int)res;
 }
+
+char *errorString(int result)
+{
+	return strerror(result);
```

Comment:
I think "curl_easy_strerror(result)" should be used since this is an error code returned by libcurl and not standard errno.